supernode/daemon/mgr/task/manager_util.go

```diff
@@ -324,6 +324,12 @@ func (tm *Manager) parseAvailablePeers(ctx context.Context, clientID string, tas
 	pieceSuccess, _ := tm.progressMgr.GetPieceProgressByCID(ctx, task.ID, clientID, "success")
 	logrus.Debugf("taskID: %s, get successful pieces: %v", task.ID, pieceSuccess)
 	if cdnSuccess && (int32(len(pieceSuccess)) == task.PieceTotal) {
+		// update dfget task status to success
+		if err := tm.dfgetTaskMgr.UpdateStatus(ctx, clientID, task.ID, types.DfGetTaskStatusSUCCESS);
+			err != nil {
```

Comment:
Don't enter a line break here. It's so weird.

Reply:
This line is too long. IMO

Reply:
Updated. PTAL.